.github/workflows/buildtest.yml

```diff
@@ -36,7 +36,7 @@ jobs:
       - name: build start
         run: |
           export ARCH=x86_64 CF="-Wsparse-error -Wsparse-all -Wno-bitwise-pointer -Wno-pointer-arith -Wno-typesign -Wnoshadow"
-          bash kconfig/kconfig-sof-default.sh
```

Comment:
Hmm @plbossart , is this intentional to remove "kconfig-sof-default.sh" run?

Reply:
@kv2019i yes it's intentional, that's how we select the IMX stuff and let Sparse check that part of the code as well.